test/version_commands.bats

```diff
@@ -5,8 +5,9 @@ load test_helpers
 setup() {
   setup_asdf_dir
   install_dummy_plugin
-  install_dummy_version "1.0.0"
-  install_dummy_version "1.1.0"
+  install_dummy_version "1.0"
+  install_dummy_version "1.1"
+  install_dummy_version "2.0"
```

Comment:
Sync versions with the versions emitted by `dummy-plugin`:

https://github.com/asdf-vm/asdf/blob/c6145d08d01691a6e16c990e7d1c8778733e8c76/test/fixtures/dummy_plugin/bin/list-all

Reply:
I actually think it would be better to sync the other way as semver test versions support any future tests around patch versions (I have an idea in the future for semver support of `asdf plugin update` which would require tests with patch versions)

Reply:
Makes sense. I've updated the branch accordingly.